builds/checkin/e2e-checkin.yaml

```diff
@@ -41,9 +41,8 @@ stages:
     condition: |
       and
       ( 
-        succeededOrFailed(),
-        in(dependencies.PublishManifests.result, 'Succeeded','Skipped'),
-        in(dependencies.BuildPackages.result, 'Succeeded','Skipped')
+        in(dependencies.PublishManifests.result, 'Succeeded','Skipped', 'SucceededWithIssues'),
+        in(dependencies.BuildPackages.result, 'Succeeded','Skipped', 'SucceededWithIssues')
```

Comment:
If one of these are skipped, will the following e2e setup still works? i.e. the PackageBuilt was skipped... now the setup doesn't have the artifact to be downloaded.

Reply:
Yes, it will work, The E2E Setup has a check to download from the ```resources.build.images``` pipeline or ```resources.build.packages``` pipeline dependening on what stage was skipped

Example : https://msazure.visualstudio.com/One/_build/results?buildId=48739070&view=results